diff --git a/gamescope/PKGBUILD b/gamescope/PKGBUILD
index 07065c858650..6ead4af06752 100644
--- a/gamescope/PKGBUILD
+++ b/gamescope/PKGBUILD
@@ -8,8 +8,7 @@ pkgdesc="gaming shell based on Xwayland, powered by Vulkan and DRM"
 arch=(x86_64)
 url="https://github.com/ValveSoftware/gamescope"
 license=('MIT')
-depends=('xorg-xwayland' 'libavif' 'aom' 'rav1e' 'libxres' 'xcb-util-errors' 'freerdp' 'xcb-util-wm' 'libxcomposite' 'pixman' 'libinput' 'seatd' 'pipewire' 'libxmu' 'libxcursor' 'libdecor' 'libei' 'luajit')
-makedepends=(openssh git meson cmake wayland-protocols ninja glslang glm vulkan-headers benchmark)
+makedepends=('git')
 source=("galileo-mura-setup.service"
         "gamescope-session"
         "gamescope-wayland.desktop"
@@ -26,15 +25,7 @@ source=("galileo-mura-setup.service"
         "steam-notif-daemon.service"
         "steam-short-session-tracker"
         "steam_http_loader.desktop"
-        "steam-http-loader"
-        "git+https://github.com/ValveSoftware/gamescope.git#tag=$_srctag"
-        "git+https://github.com/Joshua-Ashton/wlroots.git"
-        "git+https://gitlab.freedesktop.org/emersion/libliftoff.git"
-        "git+https://github.com/Joshua-Ashton/GamescopeShaders.git#tag=v0.1"
-        # FIXME Upstream gamescope is just selecting master branch at build time, so we are arbitrarily snapshotting a
-        #       revision when bumping the version here such that the build is reproducible.
-        "git+https://github.com/nothings/stb.git#commit=af1a5bc352164740c1cc1354942b1c6b72eacb8a"
-        "git+https://github.com/g-truc/glm.git#commit=0af55ccecd98d4e5a8d1fad7de25ba429d60e863")
+        "steam-http-loader")
 sha256sums=('941f5a16a495c9c4de0302819c0069749db6915345ac1e207a12544863391e4b'
             '8a9886a96a321efef5e2c928cf2e54a075b72ea37b2d12e8412fd2459f4775f2'
             'fe515fce8f151a6c03a89e043044bfddf8cd6ee89027d2cfbcf6f6706c78ca76'
@@ -61,35 +52,6 @@ sha256sums=('941f5a16a495c9c4de0302819c0069749db6915345ac1e207a12544863391e4b'
 
 install=gamescope.install
 
-prepare() {
-	cd "$pkgname"
-
-	# git submodules
-	git submodule init
-	git config submodule.subprojects/wlroots.url "$srcdir/wlroots"
-	git config submodule.subprojects/libliftoff.url "$srcdir/libliftoff"
-	git -c protocol.file.allow=always submodule update
-
-	# meson subprojects
-	rm -rf subprojects/stb
-	git clone "$srcdir/stb" subprojects/stb
-	cp -av subprojects/packagefiles/stb/* subprojects/stb/ # patch from the .wrap we elided
-
-	rm -rf subprojects/glm
-	git clone "$srcdir/glm" subprojects/glm
-	cp -av subprojects/packagefiles/glm/* subprojects/glm/ # patch from the .wrap we elided
-}
-
-build() {
-	cd "$pkgname"
-
-	rm -rf build
-	mkdir build
-	cd build
-	arch-meson --buildtype release --prefix /usr ..
-	ninja
-}
-
 package() {
 	install -D -m 755 gamescope-session "$pkgdir"/usr/lib/steamos/gamescope-session
 	install -D -m 755 steam-launcher "$pkgdir"/usr/lib/steamos/steam-launcher
@@ -115,16 +77,4 @@ package() {
 
 	# portals
 	install -D -m 644 gamescope-portals.conf "$pkgdir"/usr/share/xdg-desktop-portal/gamescope-portals/gamescope-portals.conf
-
-	install -d "$pkgdir"/usr/share/gamescope/reshade
-	cp -r "$srcdir"/GamescopeShaders/* "$pkgdir"/usr/share/gamescope/reshade/
-	chmod -R 655 "$pkgdir"/usr/share/gamescope
-
-	cd "$pkgname/build"
-
-	DESTDIR="$pkgdir" meson install --skip-subprojects
-
-	rm -rf "$pkgdir"/usr/include
-	rm -rf "$pkgdir"/usr/lib/libwlroots*
-	rm -rf "$pkgdir"/usr/lib/pkgconfig
 }
